packages/twenty-server/src/modules/messaging/blocklist-manager/listeners/messaging-blocklist.listener.ts

```diff
@@ -40,35 +47,65 @@ export class MessagingBlocklistListener {
   async handleDeletedEvent(
     payload: ObjectRecordDeleteEvent<BlocklistWorkspaceEntity>,
   ) {
-    await this.messageQueueService.add<BlocklistReimportMessagesJobData>(
-      BlocklistReimportMessagesJob.name,
-      {
-        workspaceId: payload.workspaceId,
-        workspaceMemberId: payload.properties.before.workspaceMember.id,
-        handle: payload.properties.before.handle,
-      },
+    const workspaceMemberId = payload.properties.before.workspaceMember.id;
+    const workspaceId = payload.workspaceId;
+
+    const connectedAccount =
+      await this.connectedAccountRepository.getAllByWorkspaceMemberId(
```

Comment:
it feels wrong that getAll is returning only one connectedAccount but we will change this once we have the ORM